core/spec.md

```diff
@@ -4061,10 +4063,42 @@ and the following Resource level attributes:
 
 - Constraints:
   - REQUIRED.
-  - If present, it MUST be a case sensitive value from the model defined
+  - If present, it MUST be a case-sensitive value from the model-defined
     enumeration range.
   - When not specified, the default value MUST be `none`.
   - The enumeration range MUST include `none` as a valid value.
+  - The value MAY change only when the new `compatibility` value can be
+    enforced across all existing Versions. If that's not the case, the
+    server MUST generate an error.
+
+#### `compatibilityauthority` Attribute
+- Name: `compatibilityauthority`
+- Type: Boolean
+- Description: Indicates the authority that enforces the `compatibility`
+  value defined by the owning Resource.
+
+  This specification defines the following enumeration values. Implementations
+  MAY choose to extend this list.
+  - `external` - The compatibility is enforced by an external authority.
+  - `server` - The compatibility is enforced by the server.
+
+  When set to `server`:
+  1. The server MUST reject the request if the server cannot enforce the
+     compatibility for the data to be validated for the Resource's Versions.
+  1. The server MUST reject all attempts to create/update a Resource (or its
+     Versions) that would result in those entities violating the stated
+     compatibility statement.
+  1. The server MUST reject requests to update the `compatibility`
```

Comment:
Isn't this covered by the last sentence in the previous attribute?

Reply:
I do see them as different:
1. In this case you're creating versions that may or may not be compatible, which is checked
2. In this case you're updating the compatbility setting, causing revalidation and rejecting if it can't

The intent of the user is different which I think is worth making explicit

Reply:
The sentence in the previous attribute ("compatibility") says: "The value MAY change only when the new `compatibility` value can be enforced across all existing Versions", is basically the same as:   `The server MUST reject requests to update the `compatibility` attribute value if any of the Resource's Versions would violate the compatibility rules`, isn't it?

And actually, I prefer the "The server MUST..." because the use of MAY isn't really correct because it's awkward to say "MAY only" instead of "MUST ...something...". "MAY" is for optional stuff.